SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -4,10 +4,10 @@ To allow Unreal to replicate through the SpatialOS network stack and combine mul
 
 `Dynamic Typebindings` operate at runtime so that your development iteration speed is not affected, despite your network code running on a completely different representation to Unreal’s.
 
-At the heart of `Dynamic Typebindings` is GDK-generated [SpatialOS schema]({{urlRoot}}/content/spatialos-concepts/schema), which is the SpatialOS representation of any Unreal object, its replicated data, and RPCs. `Dynamic Typebindings` also include the binding code that is invoked when the GDK converts network-relevant data between native Unreal and SpatialOS.
+At the heart of `Dynamic Typebindings` is GDK-generated [SpatialOS schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema), which is the SpatialOS representation of any Unreal object, its replicated data, and RPCs. `Dynamic Typebindings` also include the binding code that is invoked when the GDK converts network-relevant data between native Unreal and SpatialOS.
 
 # Schema
-You generate the [schema]({{urlRoot}}/content/spatialos-concepts/schema) used in `Dynamic Typebindings` via the *Schema** button in the GDK toolbar. When you select **Schema**, the GDK generates schema for all classes tagged with the [Spatial Type]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that the GDK generates schema for, there are a number of possible schema components generated, each serving a different function.
+You generate the [schema]({{urlRoot}}/content/spatialos-concepts/schema-and-snapshots#schema) used in `Dynamic Typebindings` via the *Schema** button in the GDK toolbar. When you select **Schema**, the GDK generates schema for all classes tagged with the [Spatial Type]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that the GDK generates schema for, there are a number of possible schema components generated, each serving a different function.
```

Comment:
This doesn't quite match what we've already got written up in https://docs.improbable.io/unreal/alpha/content/spatialos-concepts/schema
I don't think that page mentions Spatial Type once, and it seems like that's important information.

Reply:
Good spot! I didn't actually update this page apart from the links, so I don't have much context - do you know how we should update this in light of your schema page updates? (FYI for the schema page, I'd also made a few changes, so I had to resolve conflicts - hopefully I've incorporated all of your updates!)

Reply:
@NotRobSheridan  Is Spatial Type no longer needed for schema generation? Or is it that we didn't mention it in the schema reference doc?

Reply:
Actually - the Spatial Type doc is a bit cryptic. See Slack comment: https://improbable.slack.com/archives/CGUPE9DC4/p1557313335450100?thread_ts=1557309922.449300&cid=CGUPE9DC4

Reply:
@ElleEss I've made your suggested update (remove "all" so that it just says "generates schema for classes tagged with the Spatial Type specifier"). Do I need to do anything else in this PR about removing/explaining Spatial Type?